client/src/document/toolbar/flaws.scss

```diff
@@ -16,4 +16,9 @@
     background-color: orange;
     padding: 1px 5px;
   }
+
+  details.fixed summary,
+  li.fixed {
+    text-decoration: line-through;
```

Comment:
I still think the "Show flaws" UI looks like crap and it hurts my soul a little. But at the same time, `kumastyles` hasn't landed and Schalk has a lot on his plate.